### DISCUSSION TITLE
When will you release 3.5.2 on maven?

### DISCUSSION THREAD

**Discussion by amit3vedi** [Iterable/iterable-android-sdk#740] [General]
i can see latest SDk version is 3.5.2 but i'm not able to use 3.5.2 version in my android app because it's not available on maven. is there any timeline when you are planing to release this version?

**Comment by Ayyanchira** [comment]
Hello @amit3vedi , thanks for writing in. We re-run the process to upload to the maven and confirmed that projects can download 3.5.2 
Please try again and keep us updated. 

Thank you

**Reply by amit3vedi** [reply]
yes can download and use 3.5.2 now thanks.
